src/ORM/ArrayList.php

```diff
@@ -636,18 +636,18 @@ protected function normaliseFilterArgs($column, $value = null)
             throw new InvalidArgumentException('filter takes one array or two arguments');
         }
 
-        if (count(func_get_args()) == 1 && !is_array(func_get_arg(0))) {
+        if (count(func_get_args()) === 1 && !is_array(func_get_arg(0))) {
```

Comment:
Could use `$args = func_get_args()` and remove repeated calls to `func_get_args()`/`func_get_arg(n)`?